javaagent-tooling/src/test/groovy/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModuleTest.groovy

```diff
@@ -99,20 +93,12 @@ class InstrumentationModuleTest extends Specification {
 
   def "configure sys prop enabled for #value when default is disabled"() {
     setup:
-    Config.INSTANCE = new ConfigBuilder().readProperties([
-      "otel.instrumentation.common.default-enabled" : "false",
+    Config config = new ConfigBuilder().readProperties([
       ("otel.instrumentation." + value + ".enabled"): "true"
     ]).build()
 
-    def target = new TestInstrumentationModule([name, altName])
-    target.extend(new AgentBuilder.Default())
-
     expect:
-    target.enabled == enabled
-    target.applyCalled == enabled
-
-    cleanup:
-    Config.INSTANCE = null
+    InstrumentationModule.initEnabled(config, [name, altName], false) == enabled
```

Comment:
This also does not look like a test for `InstrumentationModule`, more like for `Config`.